From 9eb5797854146166d6e403fcc796f07b5a865507 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 4 Feb 2022 15:02:28 -0800
Subject: [PATCH 2/3] In order for binary targets to build bin must exist, so
 add it to the repo and remove it from the package's .gitignore.

---
 .gitignore     | 1 -
 bin/.gitignore | 1 +
 2 files changed, 1 insertion(+), 1 deletion(-)
 create mode 100644 bin/.gitignore

diff --git a/.gitignore b/.gitignore
index f945245..acfe23e 100644
--- a/.gitignore
+++ b/.gitignore
@@ -8,7 +8,6 @@
 *.orig
 *.test
 /pause/pause.o
-/bin/
 /test/bin2img/bin2img
 /test/checkseccomp/checkseccomp
 /test/copyimg/copyimg
diff --git a/bin/.gitignore b/bin/.gitignore
new file mode 100644
index 0000000..f935021
--- /dev/null
+++ b/bin/.gitignore
@@ -0,0 +1 @@
+!.gitignore
-- 
2.35.1

